pylint/config/utils.py

```diff
@@ -207,6 +207,7 @@ def _enable_all_extensions(run: Run, value: str | None) -> None:
     "--output": (True, _set_output),
     "--load-plugins": (True, _add_plugins),
     "--verbose": (False, _set_verbose_mode),
+    "-v": (False, _set_verbose_mode),
```

Comment:
``argparse`` permits to define both short option and long option together, I think defining both in different place is very error prone and is going to be hard to maintain in the long run.

```
>>> parser = argparse.ArgumentParser(prog='PROG')
>>> parser.add_argument('-f', '--foo')
>>> parser.add_argument('bar')
>>> parser.parse_args(['BAR'])
Namespace(bar='BAR', foo=None)
>>> parser.parse_args(['BAR', '--foo', 'FOO'])
Namespace(bar='BAR', foo='FOO')
>>> parser.parse_args(['--foo', 'FOO'])
usage: PROG [-h] [-f FOO] bar
```

https://docs.python.org/3/library/argparse.html#name-or-flags

Reply:
We already do this, but this is a different class of options. We extract them from the command line before we do any parsing and "preprocess" them. Argparse doesn't interact with these even though we define these functions as normal functions to include them in the help message.

Reply:
Do you think we could refactor the callbacks to be able to group them by short/long option ?

Reply:
Not sure what you mean? Something like `("-v", "--verbose"): (False, _set_verbose_mode)`?

Reply:
Yes, something like this where short and long options are grouped together and the callback is defined only once to both.

Reply:
(I don't know how much work the refactor would be, but it feels less error prone)

Reply:
We probably could, but I think that would be less performant? We would then need to do:
```python
for tuple in preprocessable_options:
    if option in tuple:
        do_action()
```
Instead of 
```python
if option in preprocessable_options:
    do_action()
```

It is minor, but since we only have one pre-processable action with a short flag I think it might be better to leave as is?

Reply:
Alright, your call :)